joblib/test/test_parallel.py

```diff
@@ -1285,6 +1279,27 @@ def test_parallel_unordered_generator_returns_fastest_first(backend, n_jobs):
     force_gc_pypy()
 
 
+@pytest.mark.parametrize('n_jobs', [2, 4])
+# NB: for this test to work, the backend must be allowed to process tasks
+# concurrently, so at least two jobs with a non-sequential backend are
+# mandatory.
+@with_multiprocessing
+@parametrize('backend', set(RETURN_GENERATOR_BACKENDS) - {"sequential"})
+def test_parallel_unordered_generator_returns_fastest_first(backend, n_jobs):
+    _test_parallel_unordered_generator_returns_fastest_first(backend, n_jobs)
+
+
+@pytest.mark.parametrize('n_jobs', [2, 4])
```

Comment:
I think the n_jobs parameters is just ignored when using the dask backend. So maybe we should instead test with:

```suggestion
@pytest.mark.parametrize('n_jobs', [2, -1])
```